### PR TITLE
Remove chip and kick primitives

### LINKED ISSUE #1922: Write simulated tests for ChipTactic and KickTactic
### Description of the task
Once we have simulated Tactic tests, write tests for Chip Tactic and Kick Tactic with validation
* check that robot gets behind the ball
* check that ball is kicked
* for chip: check that the ball travels over an enemy robot
* also test getbehindballtactic

### Acceptance criteria

- [ ] test chiptactic
- [ ] test kicktactic
- [ ] test getbehindballtactic

### PR DESCRIPTION
**Pull request by akhilveeraghanta**
### Description

Currently, our `chip`, `kick` and `spin` primitives don't use the trajectory planner. We don't need these additional abstractions and it will make implementing the new simulator protocol a bit simpler. 

This PR changes the kick intent to hold a move w/ autokick and the chip intent to hold a move w/ autochip. 

both the kick and chip intent are "direct control", they don't go through the navigator to replicate current behavior 

### Testing Done
All simulated tests pass, updated existing tests

The functionality is identical to what we currently have so no additional testing is required :tm:

### Resolved Issues
resolves #1922

### Length Justification
N/A

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

**Review by jonathanlew — CHANGES_REQUESTED** [review]
We should have simulated testing for KickTactic and ChipTactic to ensure the behaviour is actually the same

File: src/shared/proto/primitive.proto [hunk 1]
```diff
@@ -9,31 +9,15 @@ message Primitive
...
     {
         EstopPrimitive estop                  = 1;
-        ChipPrimitive chip                    = 2;
-        KickPrimitive kick                    = 3;
-        MovePrimitive move                    = 4;
-        SpinningMovePrimitive spinning_move   = 5;
-        StopPrimitive stop                    = 6;
-        DirectControlPrimitive direct_control = 7;
+        MovePrimitive move                    = 2;
+        SpinningMovePrimitive spinning_move   = 3;
```
Comment (jsn0716) [inline src/shared/proto/primitive.proto:13]: Is there a point in keeping this primitive in proto?

**Review by kelvinkoon — COMMENTED** [review]
Looks good, nothing to add beyond existing comments.

**Comment by jonathanlew** [comment]
I'd like to keep spinning move but that'll be migrated over to use the trajectory planner in another PR. Can you revert those changes?

File: src/shared/proto/primitive.proto (hunk 1 above)
Comment (akhilveeraghanta) [reply src/shared/proto/primitive.proto:13]: nope, but it will be removed as part of Jons PR

File: src/firmware/app/primitives/move_primitive.c [hunk 2]
```diff
@@ -36,24 +36,23 @@ typedef struct MoveState
...
-    if (time_target > 0.5f)
-    {
-        time_target = 0.5f;
-    }
-    pb->rot.time  = time_target;
-    pb->rot.vel   = pb->rot.disp / pb->rot.time;
+    pb->rot.time = (pb->min.time > TIME_HORIZON) ? pb->min.time : TIME_HORIZON;
+    // 1.4f is a magic constant to force the robot to rotate faster to its final
+    // orientation.
+    pb->rot.vel   = 1.4f * pb->rot.disp / pb->rot.time;
```
Comment (jonathanlew) [inline src/firmware/app/primitives/move_primitive.c:55]: what's behind this change?

**Review by jonathanlew — CHANGES_REQUESTED** [review]
Nice tests, just one comment

File: src/firmware/app/primitives/move_primitive.c (hunk 2 above)
Comment (akhilveeraghanta) [reply src/firmware/app/primitives/move_primitive.c:55]: ported it from `chick_motion`, basically allows the robot to rotate faster to its target orientation
without this change, the robot slowly rotates after its reached its dest which results in the kick/chip angle being incorrect

**Review by jonathanlew — DISMISSED** [review]
LGTM pending CI

**Comment by jonathanlew** [comment]
A couple more comments: add this to resolved issues https://github.com/UBC-Thunderbots/Software/issues/1922 and change the title

**Review by jonathanlew — APPROVED** [review]

**Review by jsn0716 — APPROVED** [review]
LGTM

**Pull request merged**